libkbfs/folder_branch_ops.go

```diff
@@ -837,7 +837,9 @@ func (fbo *folderBranchOps) identifyOnce(
 	ctx context.Context, md ReadOnlyRootMetadata) error {
 	fbo.identifyLock.Lock()
 	defer fbo.identifyLock.Unlock()
-	if fbo.identifyDone {
+
+	ei := getExtendedIdentify(ctx)
+	if fbo.identifyDone && !ei.behavior.AlwaysRunIdentify() {
```

Comment:
Please add a comment/TODO that this is temporary until the service sends cache-busting notices.